lib/bank_accounts/money_market_account.rb

```diff
@@ -0,0 +1,61 @@
+module BankAccounts
+
+	class MoneyMarketAccount < Account
```

Comment:
The readme.md is ambiguous about this (and this may be included in your comments of running out of time), but inheriting from SavingsAccount here helps you DRY and also gives you the functionality from the rest of Account.